go_modules/lib/dependabot/go_modules/file_parser.rb

```diff
@@ -156,6 +157,26 @@ def handle_parser_error(path, stderr)
         end
       end
 
+      GITHUB_REPO_REGEX = %r{github.com/[^@]*}.freeze
+      def handle_github_unknown_revision(line)
+        repo_path = line.scan(GITHUB_REPO_REGEX).first
+        return unless repo_path
```

Comment:
```suggestion
```

Nit: maybe too defensive? Given the guard on https://github.com/dependabot/dependabot-core/pull/2780/files#diff-5473cd4fd4f2f0f6b13968468fc3a0109a7769f9c729e4ad36a03ac8f419a660R143 this is never true.

Reply:
If we keep `line.start_with?("go: github.com/")` seems like we could get rid of it, but I don't mind the check either